src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Dates.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@using System.Security.Cryptography.X509Certificates
```

Comment:
Can you please verify is this using required. As I don't see much in the view which may require this. But I may be wrong.

Reply:
removed